packages/dev/loaders/src/STL/stlFileLoader.ts

```diff
@@ -9,6 +9,7 @@ import { registerSceneLoaderPlugin } from "core/Loading/sceneLoader";
 import { AssetContainer } from "core/assetContainer";
 import type { Scene } from "core/scene";
 import { STLFileLoaderMetadata } from "./stlFileLoader.metadata";
+import "core/Materials/standardMaterial";

```

Comment:
Is this needed because of the default material missing?

Reply:
Yes exactly. This is basically saying "as the STLFileLoader, I'm not creating any materials explicitly, so I know StandardMaterial will be used as the default material, so I will import it." This is maybe not the best for encapsulation, but I'm not sure where else we would do this. The default material is first accessed synchronously in the render loop. Alternatively, STLFileLoader could explicitly create a StandardMaterial rather than relying on the scene to provide a default material.

Reply:
I actually like the second option - never to rely on the default material. But that's out of the scope of this PR